src/common.ts

```diff
@@ -0,0 +1,15 @@
+export function gqlResultName(minBlock: number, maxBlock: number): string {
+	return `gql_result_${minBlock}-${maxBlock}.json`;
+}
+
+let _blockHeight: number | undefined;
```

Comment:
Is this just some quick in memory caching for re-use?

```suggestion
let _cachedBlockHeight: number | undefined;
```

Reply:
e3cdb0684ad89f0666d28b420f87e614303612c4